examples/sklearn-logreg-mnist/README.md

```diff
@@ -0,0 +1,61 @@
+# Flower Example using scikit-learn
+
+This example of Flower uses `scikit-learn`'s `LogisticRegression` model to train a federated learning system. It will help you understanding how to adapt Flower for use with `scikit-learn`.
```

Comment:
```suggestion
This example of Flower uses `scikit-learn`'s `LogisticRegression` model to train a federated learning system. It will help you understand how to adapt Flower for use with `scikit-learn`.
```